File: 17solve/17-205.py
a = [ int(s)
      for s in open("17-205.txt") ]

count = 0
ma = -20000;
for i in range(1,len(a)):
  if abs(a[i]-a[i-1]) % 74 == 0:
    count += 1
    ma = max( ma, a[i]+a[i-1] )

print( count, ma )

File: 17solve/17-205.pas
// Автор: А. Богданов
##
var a := ReadAllLines('17-205.txt').Select(x->x.ToInteger)
         .Toarray;

var (k,smax) := (0,0);
for var i:=0 to a.High-1 do begin
  if (a[i]-a[i+1]).Divs(37*2) then begin
    k += 1;
    sMax := max( sMax,a[i]+a[i+1] );
  end;
end;

Print( k, sMax )

